graphics.c

```diff
@@ -0,0 +1,158 @@
+#include "graphics.h"
+#include "display.h"
+
+static inline void vGraphicsPixelWrite(color pixelColor)
+{
+    vDisplayPixelWrite(pixelColor.colorSeperate.redValue, pixelColor.colorSeperate.greenValue, pixelColor.colorSeperate.blueValue);
+}
+
+
+curserStruct curserA,curserB;
+
+void graphicsInit()
+{
+
+vDisplayClear();    //blackout the display
+
+curserA.curserColor=ACURSERCOLOR;
+curserB.curserColor=BCURSERCOLOR;
+
+printGrid();    //print the grid
+
+}
+
+//printRectangle prints a Rectangle
+
+void printRectangle(uint16_t xStart,uint16_t xEnd,uint16_t yStart,uint16_t yEnd,color lineColor)
```

Comment:
- Routine in `vGraphicsDrawRect` umbenennen.
- Kommentar hinzufügen.
- X/Y-Koordinaten in eine Struktur: 
```c
typedef struct tag_tsCoord {
    uint16_t uiX;
    uint16_t uiY;
} tsCoord;
```

Reply:
Wofür steht "ts"?

Reply:
t = Typedef
s = Struktur
In diesem Fall bietet es sich sogar an, die Struktur zu packen:
```c
typedef struct __attribute__((packed)) tag_tsGraphicsCoord {
    uint16_t uiX;
    uint16_t uiY;
} tsGraphicsCoord;
```